Binner/Binner.Web/ClientApp/public/locales/de/translation.json

```diff
@@ -99,6 +99,11 @@
     "boms": {
       "header": {
         "description": "Die Stückliste ermöglicht es Ihnen, den Lagerbestand pro Projekt zu verwalten. Sie können die Mengen für jede hergestellte Leiterplatte reduzieren, und überprüfen welche Teile Sie in höherer Stückzahl kaufen müssen um die Kosten zu analysieren.<br/><br/>Wählen Sie das Projekt aus, für das Sie die Stückliste (BOM) verwalten möchten, oder erstellen Sie ein neues Projekt.<br/>" 
+      },
+      "acceptedFileTypes": "Accepted file types: \"*.xls, *.xlsx, *.csv\""
```

Comment:
just a note for the future, you don't necessarily need to add new entries for each language. If the key doesn't exist it will fall back to the english language file